packages/core/src/factory.ts

```diff
@@ -81,14 +80,22 @@ export function factory<Resource, Input, Output, Config, LoadOpts>(
         let output: Output | undefined;
         if (resource && components.mocker && (configObj as IPrismConfig).mock) {
           // generate the response
```

Comment:
`// generate the response` -> `generateTheResponse`
and later (if it must be mutated): `output = generateTheResponse`
I don't believe adding in comments justify bigger functions' bodies

Reply:
I fail to understand what you're suggesting. Can you elaborate more?

Reply:
Sure. So the code that is now marked with the comment could just go to an external function. And then we could populate `output` with the result of running this function. Maybe we could even make this `output` a `const`?

Reply:
Aaah ok I got it, let me see what we can do here!

Reply:
cool :+1:

Reply:
@XVincentX, any ETA on this?

Reply:
I'm on it now, I'm trying to see if there's a way to make it clear :)

Reply:
@lag-of-death I've tried but I haven't been able to find anything that would really satisfy me. The best I got was having two functions taking 3000 parameters anyway and not making the code any clear.

I think this needs a more in depth refactor which maybe it's not worth doing now.

If you have a specific suggestion/idea in mind I'm absolutely open; I haven't personally found anything that appealing. Let me know your ideas!

Reply:
@XVincentX, I didn't have anything specific in mind. Thanks for looking into it. Let's then have it as it is, but only for now. After what I saw, I think that refactoring `factory`/`process` deserves a separate card.